package.json

```diff
@@ -157,12 +157,12 @@
   },
   "scripts": {
     "apidocs": "node docgen/generate-docs.js",
-    "docgen:v1:extract": "api-extractor run -c docgen/api-extractor.v1.json --local",
-    "docgen:v1:gen": "api-documenter-fire markdown -i docgen/v1 -o docgen/v1/markdown && api-documenter-fire toc -i docgen/v1 -o docgen/v1/markdown/toc -p /docs/reference/functions",
-    "docgen:v1": "npm run docgen:v1:extract && npm run docgen:v1:gen",
-    "docgen:v2:extract": "api-extractor run -c docgen/api-extractor.v2.json --local",
-    "docgen:v2:gen": "api-documenter-fire markdown -i docgen/v2 -o docgen/v2/markdown && api-documenter-fire toc -i docgen/v2 -o docgen/v2/markdown/toc -p /docs/reference/functions/v2",
-    "docgen:v2": "npm run docgen:v2:extract && npm run docgen:v2:gen",
+    "docgen:v1:extract": "npm run build && api-extractor run -c docgen/api-extractor.v1.json --local",
```

Comment:
Hmmm, we probably dont want to include build in :extract and :gen - otherwise you'll have to wait for it to run build 3 times when you run `docgen:vX`

Reply:
Updated to remove the build from extract and gen.... however if someone runs `:extract` or `gen` directly they will get wonky behavior.

Reply:
But I don't have strong opinions on this too 🐢